addons/mail/models/mail_followers.py

```diff
@@ -171,11 +171,13 @@ def _get_recipient_data(self, records, message_type, subtype_id, pids=None, cids
                 query_pid = """
 SELECT DISTINCT ON (partner.id) partner.id as pid, NULL::int AS cid,
     partner.active as active, partner.partner_share as pshare, NULL as ctype,
-    users.notification_type AS notif, NULL AS groups
+    users.notification_type AS notif, array_agg(groups.id) AS groups
 FROM res_partner partner
-LEFT JOIN res_users users ON users.partner_id = partner.id AND users.active
+    LEFT JOIN res_users users ON users.partner_id = partner.id AND users.active
+    LEFT JOIN res_groups_users_rel groups_rel ON groups_rel.uid = users.id
+    LEFT JOIN res_groups groups ON groups.id = groups_rel.gid
```

Comment:
Do you really need this join ? Seems we could use directly ``groups_rel.gid``, unless there is something more ?